src/modules/UI/scenes/TransactionList/style.js

```diff
@@ -65,15 +65,33 @@ export const styles = {
   },
 
   // end of searchbar stuff
-
+  touchablBalanceBox: {
```

Comment:
please spell check your code before submitting PR's

Reply:
Good catch, although I'm not sure what the best way to implement spell-check is considering the crazy nature of variable / property names.

Reply:
PS the spelling fix has been added / committed.